root/static/scripts/tests/typeInfo.js

```diff
@@ -4785,7 +4785,7 @@ MB.relationshipEditor.exportTypeInfo({
          "cardinality1" : 0,
          "deprecated" : false,
          "id" : 281,
-         "orderableDirection" : 0,
+         "orderableDirection" : 1,
```

Comment:
This file should be automatically generated, but...meh. This is the only change we need, and it won't be reverted later, because that's how it is in the database now.